README.md

```diff
@@ -55,19 +55,28 @@
 7. Open MyGameInstance.h and include the following code:
 
 ```js
+#include "CoreMinimal.h"
+#include "Engine/GameInstance.h"
+#include "AnkrClient.h"
+#include "MyGameInstance.generated.h"
 
-   #include "AnkrClient.h"
+UCLASS()
+class MYPROJECT5_API UMyGameInstance : public UGameInstance
```

Comment:
"MYPROJECT5_API" may be modified to "MYPROJECT_API" or a generic name to indicate that the class name may be different for each user